eclipse-projects/cli-test/.classpath

```diff
@@ -43,9 +43,9 @@
       <attribute name="javadoc_location" value="jar:file:lib_managed/docs/org.codehaus.woodstox/stax2-api/stax2-api-4.1-javadoc.jar!/"/>
     </attributes>
   </classpathentry>
-  <classpathentry sourcepath="lib_managed/srcs/com.fasterxml.jackson.core/jackson-core/jackson-core-2.9.8-sources.jar" exported="true" kind="lib" path="lib_managed/bundles/com.fasterxml.jackson.core/jackson-core/jackson-core-2.9.8.jar">
```

Comment:
Did you find the automated way to maintain these classpath files?
sbt updateClassifiers
followed by running the UpdateClasspathFiles in src/main/scala in the daffodil-propgen module?

Reply:
No, I've used sed and stuff to do the replacements.  :)

Reply:
Great. For future reference, there is such a tool, but it's more important if new libraries are added or old ones removed. A version change, as you noticed, can be automated pretty easily. I did the last one with emacs tags query replace. :-)